packages/config/src/arbitrum.json

```diff
@@ -48,6 +48,10 @@
   "smol_brains_school_start_block": 3162850,
   "smol_cars_address": "0xB16966daD2B5a5282b99846B23dcDF8C47b6132C",
   "smol_cars_start_block": 3661786,
+  "smol_farm_address": "0x0000000000000000000000000000000000000000",
```

Comment:
@wyze I must have been looking at the wrong branch earlier, this has already got the 0x0 address for Smol Farm added, so it should be good to go.